website/merchandise/views.py

```diff
@@ -13,3 +13,20 @@ def index(request):
     items = MerchandiseItem.objects.all()
 
     return render(request, "merchandise/index.html", {"items": items})
+
+
+def select_product(items, id):
+    for item in items:
+        if item.id == id:
+            return item
+    return None
+
+
+def product_page(request, id):
+    items = MerchandiseItem.objects.all()
+    product = select_product(items, id)
```

Comment:
This is very inefficient. Instead, use get_object_or_404, or even better, make this view be a DetailView